content/docs/2.15/scalers/dynatrace.md

```diff
@@ -0,0 +1,93 @@
++++
+title = "Dynatrace"
+availability = "2.15+"
+maintainer = "Community"
+description = "Scale applications based on Dynatrace metric data points"
+go_file = "dynatrace_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `dynatrace` trigger that scales based on Dynatrace metric data points.
+
+```yaml
+triggers:
+  - type: dynatrace
+    metadata:
+      host: https://dummy-instance.live.dynatrace.com/
+      metricSelector: 'MyCustomEvent:filter(eq("someProperty","someValue")):count:splitBy("dt.entity.process_group"):fold'
+      # Optional
+      from: now-2d
+      threshold: "10"
+      # Optional
+      activationThreshold: "5"
+```
+
+**Parameter list:**
+
+- `host` - The Dynatrace instance to query against
+- `token` - The API key that will be leveraged to connect to Dynatrace and make requests ([official documentation](https://docs.dynatrace.com/docs/dynatrace-api/basics/dynatrace-api-authentication)). Requires the `metrics.read` scope. Must be provided via Trigger Authentication (see [Authentication Parameters](#authentication-parameters))
```

Comment:
The token can't be provided via metadata (the code is correct)
```suggestion
- `token` - The API key that will be leveraged to connect to Dynatrace and make requests ([official documentation](https://docs.dynatrace.com/docs/dynatrace-api/basics/dynatrace-api-authentication)). Requires the `metrics.read` scope. Must be provided via Trigger Authentication (see [Authentication Parameters](#authentication-parameters))
```

Reply:
correct, and that is what it says 👀

Reply:
Yeah, but we define the parameters only where they can be used (that's why host in twice for example). As `token` can be set only via TriggerAuthentication, it should be defined only in the section `### Authentication Parameters`

Reply:
And beingtotally honest, I didn't read the full message, mb 100%

Reply:
gotcha, removed 👍